src/spectacle-theme.js

```diff
@@ -276,12 +276,12 @@ export default {
   /*
    * Ecology text wrangling
    */
-  ".Overview pre": {
+  ".Ecology pre": {
```

Comment:
What relies on the `.Ecology` class? If there's nothing externally (like `formidable-landers`) that depends on it, it might make more sense to rename everything to `.Overview`. Your choice.